docs/actions.md

```diff
@@ -2933,6 +2933,100 @@ Key | Description
 
 
 
+### automatic_code_signing
+
+Updates the Xcode 8 Automatic Codesigning Flag
+
+
+
+> Updates the Xcode 8 Automatic Codesigning Flag of all targets in the project
+
+automatic_code_signing | 
+-----|----
+Supported platforms | ios, mac
+Author | @mathiasAichinger, @hjanuschka
+Returns | The current status (boolean) of codesigning after modification
+
+
+
+<details>
+<summary>8 Examples</summary>
+
+```ruby
+# enable automatic code signing
+enable_automatic_code_signing
```

Comment:
the fix for the alias thing inside docs - is included in this PR: https://github.com/fastlane/docs/pull/297